std_msgs/Header header
bool pillar_detected
bool at_pillar
float64 pillar_error
float64 distance

bool objects_detected

PerceptionObjects[] objectOfInterest
